src/jira/client/index.ts

```diff
@@ -194,13 +194,15 @@ async function getJiraClient(
 						await subscription?.update({ syncWarning: issueKeyLimitWarning });
 					}
 
-					return await batchedBulkUpdate(
+					const batchBulkUpdateReq = await batchedBulkUpdate(
```

Comment:
Updated as the pattern of `return await` is [redundant](https://eslint.org/docs/rules/no-return-await).